nextflow.config

```diff
@@ -1,4 +1,6 @@
 params {
+    gene_expression = null
+    samplesheet = null
     publish_dir_mode = "copy"
```

Comment:
I usually prefer the "link" method,

Reply:
Me too, but I would still argue for the default being `copy` as it works on all systems.

Reply:
fair enough

Reply:
I have some logic in mind to automatically determine if we can link or not, but this is beyond the scope of this project.

Reply:
It can also be solved via institutional config files (I should maybe add `publish_dir_mode='link'` to ours! 
But anyway, different discussion!